extensions/standard-processors/processors/TailFile.cpp

```diff
@@ -499,9 +499,9 @@ bool TailFile::getStateFromLegacyStateFile(const std::shared_ptr<core::ProcessCo
 }
 
 void TailFile::logState() {
-  logger_->log_info("State of the TailFile processor %s:", name_);
+  logger_->log_info("State of the TailFile processor {}:", name_);
   for (const auto& [key, value] : tail_states_) {
-    core::logging::LOG_INFO(logger_) << key << " => { " << value << " }";
+    logger_->log_info("key => {{}}", value);
```

Comment:
Did you test this line? I think a double pair of braces is just an escaped form of a pair of braces, and to actually have it contain the logger value, it would need a third pair. But I might be wrong.

Reply:
You are right, I've checked and thats the case, however I thought that these types of problems should cause compiler errors, but thats not the case apperently.

Fixed it in https://github.com/apache/nifi-minifi-cpp/commit/567d679737a20f7488cc347b4afa039d90513318